src/pseudopeople/configuration/interface.py

```diff
@@ -47,12 +49,20 @@ def get_config(dataset_name: str = None, user_config: Union[Path, str, Dict] = N
             - "taxes_w2_and_1099"
             - "women_infants_and_children"
     :param user_config: An optional override to the default configuration. Can be
-        a path to a configuration YAML file or a dictionary.
+        a path to a configuration YAML file or a dictionary. Passing a sentinel value
+        of psp.NO_NOISE ("no_noise") will override default values and return a configuration
```

Comment:
Do we want to expose the "no_noise" string here for what the sentinel value is?

Reply:
I wouldn't

Reply:
I agree, The user shouldn't know that they can type the string "no_noise" to get this functionality. They should always use `psp.NO_NOISE`. Honestly, it'd be better if the sentinel value wasn't even a string, but it's not a big deal that it is as long as we treat it as if it were an arbitrary type.